src/Nri/Css/Extra.elm

```diff
@@ -14,3 +37,115 @@ import Css exposing (..)
 lineHeightNum : Float -> Mixin
 lineHeightNum height =
     property "line-height" <| toString height
+
+
+{-| Helper for content CSS attribute
+-}
+content : String -> Mixin
+content value =
+    property "content" <| "'" ++ value ++ "'"
+
+
+{-| z-index helper
+-}
+zIndex : Int -> Mixin
+zIndex value =
+    property "z-index" <| toString value
+
+
+
+-- Cursors
+
+
+type Cursor
+    = Pointer
+    | NotAllowed
+    | Default
+    | Auto
+
+
+{-| Css cursor helper
+-}
+cursor : Cursor -> Mixin
+cursor value =
+    property "cursor" <| cursorToString value
```

Comment:
this already exists in elm-css http://package.elm-lang.org/packages/rtfeldman/elm-css/6.1.0/Css#cursor